compiler/optimizer/LocalOpts.cpp

```diff
@@ -1166,6 +1166,7 @@ int32_t TR_BlockManipulator::performChecksAndTreesMovement(TR::Block *newBlock,
          ;
 
       TR::Block *nextBlock = prevBlock->getNextBlock();
+      // If this condition is satisfied, nextBlock != NULL
```

Comment:
I added this because `nextBlock` is deferenced inside this if statement. I assumed that because of this, `nextBlock` must not be NULL at this point in the code (the warning was caused because of limitations of the static analysis tool). I added this comment for debugging purposes so someone doesn't waste time debugging an issue related to this later. I can delete it if you want.